packages/website/docs/how-does-it-work.mdx

```diff
@@ -16,44 +13,39 @@ snippet.
 
 ## You apply
 
-The first thing you'll need to do is to apply for DocSearch by [filling out the
-form on this page][1] (double check first that [you qualify][2]). We are
-receiving a lot of requests, so this form makes sure we won't be forgetting
-anyone.
+The first thing you'll need to do is to apply for DocSearch by [filling out the form on this page][1] (double check first that [you qualify][2]). We are receiving a lot of requests, so this form makes sure we won't be forgetting anyone.
 
-We guarantee that we will answer every request, but as we receive a lot of
-applications, please give us a couple of days to get back to you :)
+We guarantee that we will answer every request, but as we receive a lot of applications, please give us a couple of days to get back to you :)
 
-## We create a configuration
+## We create your Algolia application and a dedicated crawler
 
-Once we receive [your application][1], we'll have a look at your website and
-create a custom configuration file for it. This file defines which URLs we
-should crawl or ignore, as well as the specific CSS selectors to use for
-selecting headers, subheaders, etc. All configs are publicly available in our
-[config repository][3].
+Once we receive [your application][1], we'll have a look at your website, create an Algolia application and a dedicated [crawler][5] for it. Your crawler comes with [a configuration file][6] which defines which URLs we should crawl or ignore, as well as the specific CSS selectors to use for selecting headers, subheaders, etc.
 
-This step still requires some manual work and human brain, but thanks to the +1
-000 configs we already created, we're able to automate most of it. Once this
-creation finishes, we'll run a first indexing of your website and have it run
-automatically every 24 hours.
+This step still requires some manual work and human brain, but thanks to the +4 000 configs we already created, we're able to automate most of it. Once this creation finishes, we'll run a first indexing of your website and have it run automatically at a random time of the week.
```

Comment:
```suggestion
This step still requires some manual work and human brain, but thanks to the +4000 configs we already created, we're able to automate most of it. Once this creation finishes, we'll run a first indexing of your website and have it run automatically at a random time of the week.
```

Is the `+ 4 000` thing intended or a typo?

Reply:
This should be 4,000.

Reply:
Changed the suggestion for you. 😉